### PR TITLE
Fixes random in random seeds

### PR DESCRIPTION
**Pull request by ACCount12**
The lower threshold on `add_random_reagents` and `add_random_traits` wasn't set, which resulted in strange seeds being less likely to have traits and reagents than it was intended.

### CODE REVIEW

File: code/modules/hydroponics/grown/random.dm [hunk 1]
```diff
@@ -16,9 +16,9 @@
 	. = ..()
 	randomize_stats()
 	if(prob(60))
-		add_random_reagents()
+		add_random_reagents(lower=1, upper=3)
 	if(prob(50))
-		add_random_traits()
+		add_random_traits(lower=1, upper=2)
```
Comment (Cyberboss) [inline code/modules/hydroponics/grown/random.dm]: Don't use named args here

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/hydroponics/grown/random.dm (hunk 1 above)
Comment (ACCount12) [reply code/modules/hydroponics/grown/random.dm]: Why not?

**Comment by Cyberboss** [comment]
@ACCount it creates maintainability issues in that named args aren't
verified until runtime. If a proc definition changes, it can create hard to
finds bugs. Not to mention certain callstacks of ours straight don't
support it (i.e. Initialize). The only case where named args are
appropriate is when you want to use the default parameter values of some
preceding arguments.

**Comment by ACCount12** [comment]
Oh, I thought BYOND handles this compile time. Thanks for the response.

**Pull request merged**